projects/server-database/main_test.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"server-database/types"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFetchImages(t *testing.T) {
```

Comment:
so, for unit tests, you typically aren't running against a real full DB implementation. 
so you will need to use a mock DB implementation. this basically pretends to your code that it's a DB.
then you can make sure your code is behaving appropriately - check the sequence of queries that you see. 
and you can provide fake test data for your mock DB to return to your application code.
now your tests are self-contained and do not depend on a running DB with externally-provided testdata in it.

take a look at https://github.com/pashagolub/pgxmock